BusinessCentral.LinterCop/Design/Rule0072CheckProcedureDocumentationComment.cs

```diff
@@ -0,0 +1,71 @@
+﻿using BusinessCentral.LinterCop.AnalysisContextExtension;
+using Microsoft.Dynamics.Nav.CodeAnalysis;
+using Microsoft.Dynamics.Nav.CodeAnalysis.Diagnostics;
+using Microsoft.Dynamics.Nav.CodeAnalysis.Syntax;
+using Microsoft.Dynamics.Nav.CodeAnalysis.Utilities;
+using System.Collections.Immutable;
+
+namespace BusinessCentral.LinterCop.Design
+{
+    [DiagnosticAnalyzer]
+    public class Rule0072CheckProcedureDocumentationComment : DiagnosticAnalyzer
+    {
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create<DiagnosticDescriptor>(DiagnosticDescriptors.Rule0072CheckProcedureDocumentationComment);
+
+        public override void Initialize(AnalysisContext context) => context.RegisterSyntaxNodeAction(new Action<SyntaxNodeAnalysisContext>(this.AnalyzeDocumentationComments), SyntaxKind.MethodDeclaration);
+
+        private void AnalyzeDocumentationComments(SyntaxNodeAnalysisContext ctx)
+        {
+            if (ctx.IsObsoletePendingOrRemoved()) return;
+
+            var methodDeclarationSyntax = (MethodDeclarationSyntax)ctx.Node;
```

Comment:
Updated, but since i'm not that fluent in C#: Doesn't the `RegisterSyntaxNodeAction` with `SyntaxKind.MethodDeclaration` make sure we only get `MethodDeclarationSyntax`? Still makes sense to add a check if anything should change in the future.